extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -237,9 +237,18 @@ private static String minusParquetSuffix(@NotNull final String s) {
         return s;
     }
 
-    public static Function<String, String> defaultGroupingFileName(@NotNull final String path) {
-        final String prefix = minusParquetSuffix(path);
-        return columnName -> prefix + "_" + columnName + "_grouping.parquet";
+    /**
+     * Generates a grouping file path relative to the table destination file path.
```

Comment:
I thought we were re-branding "grouping" to "indexing" in this PR for new/changed code, to avoid creating a subsequent task?

Reply:
@lbooker42 mentioned that he will be doing renaming in his changes, so I didn't do it here.

Reply:
My complaint was more that we're adding more work with some of the new code, and creating a "moving target" for him.